test/metabase/driver/bigquery_test.clj

```diff
@@ -76,24 +76,37 @@
                                                                       ["field-id" (data/id :checkins :venue_id)]]]
                                                   "User ID Plus Venue ID"]]}})))
 
-;; make sure BigQuery can handle two aggregations with the same name (#4089)
```

Comment:
having the comment explaining what we're testing is still useful IMO

Reply:
That wasn't intentional, I'll put it back in 👍